src/Converter.php

```diff
@@ -310,10 +309,9 @@ private static function convertLearnosityQuestionData(array $questionDataJson)
     private static function convertLearnosityItem(array $itemJson)
     {
         // Separate question(s) and item
-        $itemJson['questionReferences'] = array_column($itemJson['questions'], 'reference');
+        $itemJson['questionReferences'] = array_column($itemJson['questions'], 'response_id');
```

Comment:
I'm just curious about this particular change - it doesn't really make sense to me for the response ID to be passed to `questionReferences` (which to me should relate to the `reference` key of a question or feature). Why was this decided on?

Reply:
Earlier we were working on wrong item json where the key response_id was present instead of reference . But later Michael told us that we need to work in the data coming from offlinpackage api.
Now we have changed it to reference .